libs/sdk-ui-dashboard/src/presentation/widget/insight/ViewModeDashboardInsight/InsightDrillDialog/DrillDialogExportDropdown.tsx

```diff
@@ -0,0 +1,239 @@
+// (C) 2021-2025 GoodData Corporation
+import React from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+import cx from "classnames";
+import {
+    BubbleHoverTrigger,
+    Bubble,
+    Dropdown,
+    UiMenu,
+    IUiMenuInteractiveItem,
+    UiMenuInteractiveItemProps,
+    DefaultUiMenuInteractiveItemComponent,
+    IAlignPoint,
+} from "@gooddata/sdk-ui-kit";
+
+import {
+    IExecutionResultEnvelope,
+    selectExecutionResultByRef,
+    selectSettings,
+    useDashboardSelector,
+} from "../../../../../model/index.js";
+import { idRef } from "@gooddata/sdk-model";
+import { isDataError, isDataErrorTooLarge } from "../../../../../_staging/errors/errorPredicates.js";
+
+export interface IDrillDialogShareDropdownProps {
+    exportAvailable: boolean;
+    exportXLSXEnabled: boolean;
+    exportCSVRawEnabled: boolean;
+    onExportXLSX: () => void;
+    exportCSVEnabled: boolean;
+    onExportCSV: () => void;
+    onExportCSVRaw: () => void;
+    isLoading: boolean;
+    isExporting: boolean;
+    isExportRawVisible: boolean;
+}
+
+const dropdownDisabledBubbleAlignPoints: IAlignPoint[] = [{ align: "tc bc" }];
+
+interface IMenuItemData {
+    interactive: {
+        action: () => void;
+        disabledTooltip?: string;
+        className?: string;
+    };
+}
+type IMenuItem = IUiMenuInteractiveItem<IMenuItemData>;
+
+const DRILL_MODAL_EXECUTION_PSEUDO_REF = idRef("@@GDC_DRILL_MODAL");
+
+const getExportTooltip = (execution?: IExecutionResultEnvelope, enableRawExports?: boolean): string => {
+    if (isDataErrorTooLarge(execution?.error)) {
+        return "options.menu.data.too.large";
```

Comment:
use import { defineMessages } from "react-intl"; for definition of all lang IDs from component. It enables lang IDs validation